package.json

```diff
@@ -18,6 +19,7 @@
     "build:cjs": "esbuild src/js/*.js src/js/*/*.js src/js/*/*/*.js --target=es2019 --format=cjs --outdir=dist/cjs",
     "build:iife": "esbuild src/js/index.js --bundle --target=es2019 --format=iife --outdir=dist/iife --minify --sourcemap --global-name=MediaChrome && yarn size",
     "build:react": "node ./scripts/react/build.js",
+    "build:manifest": "cem analyze --config scripts/custom-elements-manifest.config.js",
     "build": "yarn build:esm && yarn build:cjs && yarn build:iife && yarn build:react && yarn build:types",
```

Comment:
suggestion(non-blocking): Update `build` script to include `build:manifest`

Reply:
will do